concepts/58 jQuery Components/05 Add DevExtreme to a jQuery Application/10 Local Files.md

```diff
@@ -1,19 +1,25 @@
 
     <!--HTML-->
-    <script type="text/javascript" src="js/jquery-3.4.1.min.js"></script>
+    <head>
+        <!-- ... -->
+        <script type="text/javascript" src="js/jquery-3.4.1.min.js"></script>
 
-    <!-- DevExtreme themes -->
-    <link rel="stylesheet" href="css/dx.common.css">
-    <link rel="stylesheet" href="css/dx.light.css">
+        <!-- DevExtreme themes -->
+        <link rel="stylesheet" href="css/dx.common.css">
+        <link rel="stylesheet" href="css/dx.light.css">
 
-    <!-- DevExtreme library -->
-    <script type="text/javascript" src="js/dx.all.js"></script>
-    <!-- <script type="text/javascript" src="js/dx.web.js"></script> -->
-    <!-- <script type="text/javascript" src="js/dx.viz.js"></script> -->
-    <!-- <script type="text/javascript" src="js/dx.viz-web.js"></script> -->
+        <!-- DevExtreme library -->
+        <script type="text/javascript" src="js/dx.all.js"></script>
+        <!-- <script type="text/javascript" src="js/dx.web.js"></script> -->
+        <!-- <script type="text/javascript" src="js/dx.viz.js"></script> -->
+        <!-- <script type="text/javascript" src="js/dx.viz-web.js"></script> -->
+    </head>
+    <body class="dx-viewport">
+        <!-- ... -->
+    </body>
 
-The stylesheets in the code above apply the Generic Light theme. See [Predefined Themes](/concepts/60%20Themes%20and%20Styles/05%20Predefined%20Themes/00%20Predefined%20Themes.md '/Documentation/Guide/Themes_and_Styles/Predefined_Themes/') for information about the other available themes.
+The stylesheets in the code above apply the Generic Light theme. The `dx-viewport` class ensures that theme colors and typography settings are applied to all elements on the page, not only to DevExtreme widgets. See [Predefined Themes](/concepts/60%20Themes%20and%20Styles/05%20Predefined%20Themes/00%20Predefined%20Themes.md '/Documentation/Guide/Themes_and_Styles/Predefined_Themes/') for information about other available themes.
 
 Specific functionality requires additional libraries. For instance, [client-side exporting](/concepts/05%20Widgets/DataGrid/70%20Client-Side%20Exporting '/Documentation/Guide/Widgets/DataGrid/Client-Side_Exporting/') in the **DataGrid** and **PivotGrid** widgets requires the <a href="https://stuk.github.io/jszip/" target="_blank">JSZip</a> library. Similarly, the [HtmlEditor](/concepts/05%20Widgets/HtmlEditor/00%20Overview.md '/Documentation/Guide/Widgets/HtmlEditor/Overview/') widget requires <a href="https://quilljs.com/" target="_blank">Quill</a>. Refer to the dedicated topics for details.
```

Comment:
In the first sentence, change "in the **DataGrid** and **PivotGrid** widgets" to "in **DataGrid** and **PivotGrid** widgets"

If the last sentence is referring to links in the previous sentences, can we delete this last sentence (as it would be redundant)?